docs/MAINTAINERS.adoc

```diff
@@ -161,18 +163,33 @@ Code reviews do NOT pursue the following goals:
   or in general when we expect at least a large minority of admins to benefit from knowing about the change, e.g. to apply a new option.
 * It would make sense to backport the change to LTS, a Jira issue should exist and be labeled as `lts-candidate`
 
-**Step 2: Marking for merge**. Once the checklist is passed,
-a Core PR Reviewer or a Maintainer can mark the pull request for merge.
+==== Step 2: Is it a good time to merge?
+
+link:[https://jenkins.io/security/](Jenkins security updates) are coordinated with the LTS calendar, and if the weekly introduces regressions, users of that line may have to choose between security fixes, and a working Jenkins.
+For these reasons, the following changes should not be merged during the week before LTS releases (weeks 3, 7, 11, 15, etc. on the page linked above):
+
+* Changes that could be considered risky (relatively high risk of introducing regressions), as they could make users of Jenkins weekly releases choose between getting security fixes, and having a functioning Jenkins
+* Very large changes (in terms of lines changed), because the Jenkins security team needs to prepare security fixes for the weekly release line in a very short period of time.
+
+If the change is ready but it is not a good time, consider labeling the pull request with the `on-hold` label.
+If a merge is needed, please consult with the Jenkins Security officer
```

Comment:
No such thing?

Reply:
Hm? Do you suggest removing it?

Reply:
I'm questioning the premise of "a merge is needed".

Should have submitted your PR earlier if you need something very large to go into a specific weekly release.

Reply:
I will just remove it for now. There are always ways to escalate of merge is needed